app/script.rb

```diff
@@ -0,0 +1,184 @@
+require 'pry'
+require 'watir'
+require 'nokogiri'
+require 'json'
+require 'date'
+
+class VB_WebBanking
+  BASE_URL = "https://web.vb24.md/wb/".freeze
+  ACCOUNTS_URL = "#{BASE_URL}#menu/MAIN_215.NEW_CARDS_ACCOUNTS".freeze
+  TRANSACTIONS_URL = "#{BASE_URL}#menu/MAIN_215.CP_HISTORY".freeze
+
+  attr_reader :accounts, :transactions
+
+  def run
+    browser.goto(BASE_URL)
+    sleep 2
+    authentication_check
+    check_accounts
+    check_transactions
+    store
+  end
+
+  private
+  
+  def browser
+    @browser ||= Watir::Browser.new :chrome
+  end
+
+  def authentication_check
+    if browser.text_field(name: "login").present?
+      puts "Authentication required"
+      authentication
+    end
+  end
+
+  def authentication
+    unless File.exist?('data/login.json')
+      manual_login
+    else
+      local_login
+    end
+    browser.button(class: "wb-button").click
+    sleep 2
+    if browser.div(class: "block__cards-accounts").exist?
+      puts "Authentication successful"
+    else
+      puts "Authentication failed, try again"
+      manual_login
+    end
+  end
+
+  def local_login
+    file = File.read('data/login.json')
+    json = JSON.parse(file)
+    browser.text_field(name: "login").set(json["login"])
+    browser.text_field(name: "password").set(json["password"])
+  end
+
+  def manual_login
+    puts "Enter your login"
+    browser.text_field(name: "login").set(gets.chomp)
```

Comment:
Review all your code and separate it with blank lines by the logical blocks. like:
```ruby
def manual_login
  puts "Enter your login"
  browser.text_field(name: "login").set(gets.chomp)

  puts "Enter your password"
  browser.text_field(name: "password").set(gets.chomp)
...
```
This way the code is more readable. 
If you write a set of variables to use them later on, also pile them up but separate from the following logic:
```ruby
data     = some_data
variable = some_variable
numbers  = set_of_numbers

if numbers.include?("2") 
...
```